sda-commons-server-mongo-testing/src/test/java/org/sdase/commons/server/mongo/testing/UseExistingMongoDbRuleTest.java

```diff
@@ -24,12 +25,13 @@ public class UseExistingMongoDbRuleTest {
           .withPassword("testpassword")
           .build();
 
-  private static SystemPropertyRule SYSTEM_PROPERTY_RULE;
+  @ClassRule public static SystemPropertyRule SYSTEM_PROPERTY_RULE;
 
   private MongoDbRule useExistingMongoDbRule;
 
   @BeforeClass
   public static void initEnvAfterStartOfExternalDb() {
+    ((StartLocalMongoDbRule) EXTERNAL_DB).before();
```

Comment:
Why do we need to invoke the method manually? EXTERNAL_DB is also defined as `ClassRule`